app/models/container_image.rb

```diff
@@ -36,6 +36,7 @@ class ContainerImage < ApplicationRecord
   after_create :raise_creation_event
 
   def full_name
+    docker_id if image_ref && image_ref.start_with?(DOCKER_PULLABLE_PREFIX)
```

Comment:
This looks like a no-op line to me.  It will evaluate `docker_id`, but `docker_id` isn't actually used in this method anywhere.

Did you mean to `return docker_id`?

EDIT:

The best part is I merged this same change upstream...